pkg/test/resourcefixture/contexts/alloydb_context.go

```diff
@@ -25,4 +25,14 @@ func init() {
 		ResourceKind: "AlloyDBBackup",
 		SkipUpdate:   true,
 	}
+
+	resourceContextMap["restorebackupalloydbcluster"] = ResourceContext{
+		ResourceKind: "AlloyDBCluster",
+		SkipUpdate:   true,
+	}
+
+	resourceContextMap["zonalalloydbinstance"] = ResourceContext{
+		ResourceKind: "AlloyDBInstance",
+		SkipUpdate:   true,
```

Comment:
Are we skipping update because we want to accelerate the test run? Or because update is not supported?

Reply:
Because we want to accelerate the test run, as we already have tried updates for other test cases for AlloyDBCluster